ftxwebsocket/websocket_manager.py

```diff
@@ -58,11 +59,11 @@ def wrapped_f(ws, *args, **kwargs):
 
     def _run_websocket(self, ws):
         try:
-            ws.run_forever()
+            ws.run_forever(ping_timeout=2, ping_interval=5)
```

Comment:
Pass `ping_timeout` and `ping_interval` arguments to ensure that `_on_error` handlers are triggered when the remote host (FTX) becomes unreachable. See https://websocket-client.readthedocs.io/en/latest/app.html?highlight=run_forever#websocket._app.WebSocketApp.run_forever.